index.html

```diff
@@ -5860,107 +5860,71 @@ <h3>Vocabularies</h3>
-Recommendation phase based on migration of documents to the W3C Technical
-Reports namespace and implementer feedback that requires the referenced URLs to
-be modified.
+Recommendation phase based on migration of documents to time-stamped locations,
+migration of documents to the W3C Technical Reports namespace, and/or
+implementer feedback that requires the referenced URLs to be modified.
         </p>
 
         <p>
-Implementations MUST ensure that the following vocabulary URLs used in the base
-context ultimately resolve to the following files, which are normative:
+Implementations that depend on RDF vocabulary processing MUST ensure that the
+following vocabulary URLs used in the base context ultimately resolve to the
+following files, which are normative. Other semantically equivalent
+serializations of the vocabulary files MAY be utilized by implementations.
+Cryptographic hashes are provided for all content to ensure that developers can
+verify that the contents of each file are correct.
         </p>
 
         <table class="simple">
           <thead>
             <tr>
-              <th>URL</th>
-              <th>Media Type</th>
-              <th>Content</th>
+              <th>URL and Media Type</th>
+              <th>Content and Hashes</th>
             </tr>
           </thead>
           <tbody>
             <tr>
               <td>
-https://www.w3.org/2018/credentials#
-              </td>
-              <td>
-text/html
-              </td>
-              <td>
-https://www.w3.org/2018/credentials/index.html
-              </td>
-            </tr>
-            <tr>
-              <td>
-https://www.w3.org/2018/credentials#
-              </td>
-              <td>
+https://www.w3.org/2018/credentials#<br>
 application/ld+json
               </td>
               <td>
-https://www.w3.org/2018/credentials/index.jsonld
+https://www.w3.org/2018/credentials/index.jsonld<br><br>
+sha256: z52TgKqh2nqTCuACI8lCvhRdjwxQjeVmuOMCDCEijq4=<br><br>
+sha3-512: m8Ss+jgZiyL2Ws/ICJcWjHFd9PccJWsXPvMatBOhrH<wbr>
+h0qCBrzfgO2zO1OQQbTL7zoPgLseIbcxJJpunD2bkoRA==
               </td>
             </tr>
             <tr>
               <td>
-https://schema.org/
-              </td>
-              <td>
-text/html
-              </td>
-              <td>
-https://schema.org/
-              </td>
-            </tr>
-            <tr>
-              <td>
-https://schema.org/
-              </td>
-              <td>
+https://schema.org/<br>
 application/ld+json
               </td>
               <td>
-https://schema.org/version/latest/schemaorg-current-https.jsonld
-              </td>
-            </tr>
-            <tr>
-              <td>
-https://w3id.org/security#
-              </td>
-              <td>
-text/html
-              </td>
-              <td>
-https://w3c.github.io/vc-data-integrity/vocab/security/vocabulary.html
+https://schema.org/version/22.0/schemaorg-all-https.jsonld<br><br>
```

Comment:
does this mean that our use of `name` and `description` are normatively bound to version 22.0 of schema.org ? and if version 23.5 changes those term definitions verifiable credentials using the new definitions are not conformant?

Reply:
It means when you evaluate `name` and `description`, you are to use the v22.0 meaning of those properties.

This is the downside with using cryptographic hashes and specific releases, you don't end up using the latest meaning of the vocabulary. That said, I'd be surprised if `name` and `description` changed in any significant way in future releases.

Reply:
It seems like more of a downside that comes from using URLs for context files, we hope version 22.0 won't change terms, and that version 23.0 won't add terms we need, we have a way to detect if version 22 changes terms, and we have caching guidance from the remote server that applies to this.

I think we have done the best we can for URLs that reference context files, but if this thread motivates additional improvements mission accomplished.

Reply:
@msporny too close off this thread, do we need to add any updates regarding normative dependency on a specific version, or comment on caching headers? We had a thread somewhere else about how long to cache the w3c hosted v2 context, that is also relevant here.

Reply:
My feeling is that we are overzealous at this point. We are using some absolute base terms of schema.org that is used at millions of places all over the world. The chances for changes are practically zero. In my view we can safely leave out schema.org from this whole consideration. After all, we also refer, indirectly, to xsd datatypes, rdf and rdfs core terms, and we do not put hash values for their respective URL-s either.